prime-router/docs/design/proposals/0024-original-message-passthrough.md

```diff
@@ -0,0 +1,172 @@
+# Proposal for sendOriginal Pipeline Function
+
+## Background
+
+In order to meet data integrity requirements, a sender has requested a feature in the Universal Pipeline to
+preserve the original unaltered message and convey it to receivers with no alterations, including preserving the
+original file name.
+
+## Assumptions
+
+* While this design is intended to support a specific use case, CDC ELIMS ETOR, it should be easily applicable to other
+  use cases as well and fit into the existing architecture of the pipeline.
+* Senders will send using a message format supported by the Universal Pipeline (currently HL7v2 and FHIR).
+* Senders using the sendOriginal functionality shall NOT send batched messages. Each submitted report shall contain only
```

Comment:
no changes required here, just pointing out this assumption should be verified by the pipeline in the implementation and the report should be rejected if it contains more than one message.